content/v2.0/reference/api/python-cl-guide.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Python client library
+description: >
+  Use the Python client library to interact with InfluxDB
+weight: 103
+menu:
+  v2_0_ref:
+    name: Python client library
+    parent: InfluxDB v2 API
+v2.0/tags: [client libraries]
+---
+
+Use the [InfluxDB Python client libary](https://github.com/influxdata/influxdb-client-python) to integrate InfluxDB into Python scripts and applications.
+
+This guide presumes some familiarity with Python and InfluxDB.
+If you haven't, go ahead and read the [getting started](/v2.0/get-started/) guide.
+
+## Before you begin
+
+1. Install the InfluxDB Python library by running
+
+    ```sh
+    pip install influxdb-client
+    ```
+
+2. Ensure that InfluxDB is running.
```

Comment:
Rather than "Ensure InfluxDB is running," they need to get their InfluxDB API url. A user can use the client to interact with InfluxDB Cloud or a local InfluxDB instance. Which one they're using determines the URL. This should point them to where to get it.

If running InfluxDB locally, it's `http://localhost:9999`. If using InfluxDB Cloud, it's the URL of their InfluxDB Cloud UI. For example: `https://us-west-2-1.aws.cloud2.influxdata.com`.